man/spark_read_json.Rd

```diff
@@ -14,6 +14,8 @@ spark_read_json(sc, name, path, repartition = 0, memory = TRUE,
 
 \item{path}{The path to the file. Needs to be accessible from the cluster. Supports: "hdfs://" or "s3n://"}
 
+\item{options}{A list of strings with additional options.}
```

Comment:
Added reference for parquet. Haven't found an authoritative reference for JSON yet. https://github.com/rstudio/sparklyr/commit/d6d1a1c6ab8599e5abfd753eab4a8ea85ceaee7e